index.js

```diff
@@ -40,27 +40,31 @@ function initialize(options) {
     sdkKey,
     datafile,
     logLevel,
-    datafileOptions,
   } = options;
 
 
   const defaultLogger = require('@optimizely/optimizely-sdk').logging;
-  const manager = new DatafileManager({
-    sdkKey,
+
+  const optimizelyClient = OptimizelySdk.createInstance({
```

Comment:
FYI, the only potential drawback of this approach is that during the lifetime of a single request to the server the `optimizelyClient` may return different values for the feature if the datafile is updated while a request is being handled.

It was a design choice originally to try and ensure the client always evaluated a feature flag or experiment the same throughout the lifetime of a single request, which is why the datafile was passed directly and a client instance was created each time at the beginning of the request [here](https://github.com/optimizely/express-middleware/blob/master/index.js#L77). But we'd be curious if you think this is enough to warrant using the alternative fix here: https://github.com/optimizely/express-middleware/pull/7